doc/guide/sesans/sans_to_sesans.rst

```diff
@@ -0,0 +1,31 @@
+.. currentmodule:: sasmodels
+.. Wim Bouwman, DUT, written at codecamp-V, Oct2016
+
+.. _SESANS:
+
+SANS to SESANS conversion
+=========================
+
+The conversion from SANS into SESANS in absolute units is a simple Hankel
+transformation when all the small-angle scattered neutrons are detected.
+First we calculate the Hankel transform including the absolute intensities by
+
+.. math:: G(\delta) = 2 \pi \int_0^{\infty} J_0(Q \delta) \frac{d \Sigma}{d \Omega} (Q) Q dQ \!,
+
+in which :math:`J_0` is the zeroth order Bessel function, :math:`\delta`
+the spin-echo length, :math:`Q` the wave vector transfer and :math:`\frac{d \Sigma}{d \Omega} (Q)`
+the scattering cross section in absolute units. This is a 1-dimensional
+integral, which can be rather fast. In the numerical calculation we integrate
+from :math:`Q_{min} = 0.1 \times 2 \pi / R_{max}` in which :math:`R_{max}`
+will be model dependent. We determined the factor 0.1 by varying its value
```

Comment:
I just added some changes to try and clear up the language.  This also brought to light that the sasview and sasmodels Sesans code are out of sync.  I'll get a pull request out for that immediately.